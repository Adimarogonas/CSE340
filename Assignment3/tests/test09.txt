{
    r = + s 2.4;
    if (s)
    {
        r = 0.00001;
    }
}
