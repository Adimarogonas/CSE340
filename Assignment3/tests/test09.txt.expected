TYPE MISMATCH 3 C4
